x/superfluid/keeper/migrate_test.go

```diff
@@ -797,13 +797,14 @@ func (s *KeeperTestSuite) TestValidateMigration() {
 	}
 }
 
-func (s *KeeperTestSuite) TestValidateSharesToMigrateUnlockAndExitBalancerPool() {
+func (s *KeeperTestSuite) TestForceUnlockAndExitBalancerPool() {
 	defaultJoinTime := s.Ctx.BlockTime()
 	type sendTest struct {
 		overwritePreMigrationLock bool
 		overwriteShares           bool
 		overwritePool             bool
 		overwritePoolId           bool
+		exitCoinsLengthIsTwo      bool
 		percentOfSharesToMigrate  sdk.Dec
```

Comment:
Lets remove percentOfSharesToMigrate as a param now that this is a fixed value.

Reply:
Sorry I don't think I understood you correctly, ForceUnlockAndExitBalancerPool still supports partials right?